src/js/stores/MesosSummaryStore.js

```diff
@@ -178,6 +178,42 @@ class MesosSummaryStore extends GetSetBaseStore {
     });
   }
 
+  getFrameworkTasksByName(serviceName = '') {
```

Comment:
Do we maybe want to move this to a Util? We have something _very_ similar in `MesosStateStore`